Include/fileutils.h

```diff
@@ -117,7 +117,15 @@ PyAPI_FUNC(int) _Py_dup(int fd);
 PyAPI_FUNC(int) _Py_get_blocking(int fd);
 
 PyAPI_FUNC(int) _Py_set_blocking(int fd, int blocking);
-#endif   /* !MS_WINDOWS */
+#else   /* MS_WINDOWS */
+PyAPI_FUNC(intptr_t) _Py_get_osfhandle_noraise(int fd);
```

Comment:
I thought one of benefits would be using `HANDLE` in the signature instead of `intptr_t`, to avoid all of the casts.